rootfs/scheduler/__init__.py

```diff
@@ -936,62 +936,6 @@ def delete_rc(self, namespace, name):
 
         return response
 
-    def _healthcheck(self, namespace, container, routable=False, path='/', port=5000,
```

Comment:
Yes, it sure is and I think the serializer should still do the validation but I was mostly wondering given how we consider the scheduler a separate module that maybe serializer should pull in "validation" from the scheduler plus we may have a different API vs Scheduler structure in the health check department, either due to differences in k8s versions or changes in internal design.

Trying to make sure we don't cause us debt later :) I'm not fully sold on any of the above ideas particularly, yet.